server/etcdmain/help.go

```diff
@@ -273,7 +273,7 @@ Experimental distributed tracing:
     Number of samples to collect per million spans for distributed tracing. Disabled by default.
 
 Experimental feature:
-  --experimental-initial-corrupt-check 'false'
+  --experimental-initial-corrupt-check 'false'.It's deprecated, and will be decommissioned in v3.7. Use '--feature-gates=InitialCorruptCheck=true' instead.
```

Comment:
nit: add a space after the .
```suggestion
  --experimental-initial-corrupt-check 'false'. It's deprecated, and will be decommissioned in v3.7. Use '--feature-gates=InitialCorruptCheck=true' instead.
```